File: fixtures/scale/kotlin/medium/src/calc/app/Installer.kt
package calc.app

import calc.config.Settings
import calc.eval.FunctionRegistry
import calc.funcs.registerCombinatorics
import calc.funcs.registerConversion
import calc.funcs.registerExponential
import calc.funcs.registerHyperbolic
import calc.funcs.registerInverseTrig
import calc.funcs.registerLogarithms
import calc.funcs.registerPowers
import calc.funcs.registerRounding
import calc.funcs.registerStats
import calc.funcs.registerTrig

fun installBuiltins(registry: FunctionRegistry, settings: Settings) {
    registerTrig(registry, settings)
    registerInverseTrig(registry, settings)
    registerHyperbolic(registry)
    registerExponential(registry)
    registerLogarithms(registry)
    registerPowers(registry)
    registerRounding(registry)
    registerStats(registry)
    registerCombinatorics(registry)
    registerConversion(registry)
}

File: fixtures/scale/dart/medium/lib/app/installer.dart
import '../config/settings.dart';
import '../eval/function_registry.dart';
import '../funcs/combinatorics.dart';
import '../funcs/conversion.dart';
import '../funcs/exponential.dart';
import '../funcs/hyperbolic.dart';
import '../funcs/inverse_trig.dart';
import '../funcs/logarithms.dart';
import '../funcs/powers.dart';
import '../funcs/rounding.dart';
import '../funcs/stats.dart';
import '../funcs/trig.dart';

void installBuiltins(FunctionRegistry registry, Settings settings) {
  registerTrig(registry, settings);
  registerInverseTrig(registry, settings);
  registerHyperbolic(registry);
  registerExponential(registry);
  registerLogarithms(registry);
  registerPowers(registry);
  registerRounding(registry);
  registerStats(registry);
  registerCombinatorics(registry);
  registerConversion(registry);
}
